setup.py

```diff
@@ -26,13 +28,55 @@
 LONG_DESCRIPTION = readme
 AUTHOR = "Facebook AI Research"
 LICENSE = license
-REQUIREMENTS = (reqs.strip().split("\n"),)
+REQUIREMENTS = reqs.strip().split("\n")
+DEFAULT_EXCLUSION = ["test", "examples"]
+
+FULL_REQUIREMENTS = []
+# collect requirements.txt file in all subdirectories
+for file_name in glob.glob("**/*requirements.txt", recursive=True):
+    with open(file_name) as f:
+        reqs = f.read()
+        FULL_REQUIREMENTS.extend(reqs.strip().split("\n"))
+
+
+# overriding install and develop
+class OptionedCommand(object):
+    user_options = [("all", None, "include habitat_baselines in installation")]
+
+    def initialize_options(self):
+        super().initialize_options()
```

Comment:
This is bad practice. if someone inherited `OptionedCommand` without inheriting relevant module, this will crash, let's inherit proper class here.

Reply:
Discussed with Jiazhi. For now, since there is no way to go around this, we can keep this. Most of the code that I looked at which does similar kind of thing, actually does their extra stuff in common functions.

Reply:
Agree with @apsdehal, let's add a verbose comment, regarding what's happening here.